src/yoroi-wallets/cardano/utxoManager.ts

```diff
@@ -75,3 +77,61 @@ export const makeUtxoStorage = (storage: typeof storageLegacy, storagePath: stri
 
   return utxoStorage
 }
+
+const makeUtxoManagerStorage = (storage: typeof storageLegacy, storagePath: string) => {
```

Comment:
to persist the state, the anatomy right now is `utxoManager/addCounter` -> number